<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pos0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pos3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>489</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLACEHOLDER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTic_Tac_Toe">
    <property name="title">
     <string>Tic Tac Toe</string>
    </property>
   </widget>
   <addaction name="menuTic_Tac_Toe"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
